tasks/__init__.py

```diff
@@ -85,6 +85,11 @@ def adminserver(port=8001):
     cmd = '{} python manage.py runserver {} --nothreading'.format(env, port)
     run(cmd, echo=True, pty=True)
 
+@task
+def admin_task(args):
```

Comment:
This is redundant with `inv admin.manage`, a task the existence of which is [hinted](https://github.com/CenterForOpenScience/osf.io/tree/efb51ffac31fe79a0ad508dded1764e3100bf8fd/admin#how-to-run-the-admin-interface) at in the README.